Compiler error build using 0.3.0-alpha.1

Trying to run a binary app with dependency on `build 0.3.0-alpha.1` on windows with `cargo 1.39.0-nightly (22f7dd049 2019-08-27)` has a compiler error:
```
error[E0599]: no method named `poll_next` found for type `std::pin::Pin<&mut tokio_sync::mpsc::bounded::Receiver<message::Message<Request, <T as tower_service::Service<Request>>::Future>>>` in the current s
cope
   --> C:\Users\Mac\.cargo\registry\src\github.com-1ecc6299db9ec823\tower-buffer-0.3.0-alpha.1\src\worker.rs:118:65
    |
118 |         while let Some(mut msg) = ready!(Pin::new(&mut self.rx).poll_next(cx)) {
    |                                                                 ^^^^^^^^^ help: there is a method with a similar name: `poll_recv`

error: aborting due to previous error
```

Built with this Cargo.toml:
```
[package]
name = "test-tower"
version = "0.1.0"
authors = ["Mackenzie Clark <mackenzie.a.z.c@gmail.com>"]
edition = "2018"

[dependencies]
tower = "0.3.0-alpha.1"
```

An empty main function:
```
fn main() {
}
```

--- a/tower-buffer/src/worker.rs
+++ b/tower-buffer/src/worker.rs
@@ -115,7 +115,7 @@ where
         }
 
         // Get the next request
-        while let Some(mut msg) = ready!(Pin::new(&mut self.rx).poll_next(cx)) {
+        while let Some(mut msg) = ready!(Pin::new(&mut self.rx).poll_recv(cx)) {
             if msg.tx.poll_closed(cx).is_pending() {
                 tracing::trace!("processing new request");
                 return Poll::Ready(Some((msg, true)));

